<launch>
    <node pkg="apb_controllers" name="serial_to_mcu_apb" type="serial_to_mcu_apb" output="screen">
      <param name="serial_port" type="string" value="/dev/ttyUSB0" />
      <param name="baudrate" type="int" value="115200" />
    </node>
</launch>
